### DISCUSSION TITLE
Is it possible to add simple animated water visualization using luma.gl and deck.gl?

### DISCUSSION THREAD

**Discussion by am2222** [visgl/deck.gl#9182] [Q&A]
Hi,
Is it possible to visualize animated water on the map using a custom deckgl layer?
I have done some research and it seems there is ways we can visualize water using mapbox and threebox. however it needs to create a large plane and cover entire map with that plane and then render the land on top of it. However with deck.gl I noticed we can have sublayers for MVT layers and fill-extension . I also noticed `luma.gl` supports 3d textures (https://luma.gl/examples/api/texture-3d/) so was wondering if its possible to have a water like texture?

**Comment by Ayman-SIDQI** [comment]
Hi @am2222 did you manage to figure it out?

**Reply by am2222** [reply]
@Ayman-SIDQI no unfortunately!
